src/objects/domain_obj.f90

```diff
@@ -936,17 +936,18 @@ subroutine setup_sleve(this, options)
             zr_u                  => this%zr_u,                           &
             zr_v                  => this%zr_v)
 
-            ! Still not 100% convinced this works well in cases other than flat_z_height = 0 (w sleve). So for now best to keep at 0 when using sleve?
+            
             max_level = find_flat_model_level(options, nz, dz)
 
-            if(max_level /= nz) then
-                if (this_image()==1) then
-                    print*, "    flat z height ", options%parameters%flat_z_height
-                    print*, "    flat z height set to 0 to comply with SLEVE coordinate calculation "
-                    print*, "    flat z height now", nz
-                end if
-                max_level = nz
-            end if
+            ! Still not 100% convinced this works well in cases other than flat_z_height = 0 (w sleve). So for now best to keep at 0 when using sleve?
+            ! if(max_level /= nz) then
+            !     if (this_image()==1) then
+            !         print*, "    flat z height ", options%parameters%flat_z_height
+            !         print*, "    flat z height set to 0 to comply with SLEVE coordinate calculation "
+            !         print*, "    flat z height now", nz
+            !     end if
+            !     max_level = nz
+            ! end if
 
             smooth_height = sum(dz(1:max_level)) !sum(global_terrain) / size(global_terrain) + sum(dz(1:max_level))
```

Comment:
thanks, similarly, is there a reason the global_terrain bit is commented out?  That seems like a good thing to keep in there?

Reply:
That would lead to less compression in the layers - which we could decide to do, it would make the actual dz thickness be closer to the namelist dz thickness. But this way it is consistent with the non-SLEVE formulation.

Reply:
It would also probably require people to increase their decay rates to achieve the same grid structure as before. So if we change it we should announce that clearly.

Reply:
OK, maybe don't do anything for it at the moment, we could also probably use minval(global_terrain), so when the domain intersects sea level, it is identical to current, but if the lowest point in the domain is 2000m, you get a better(?) behavior on lower levels?